### PR TITLE
0.12.7 docs/examples

### PR DESCRIPTION
**Pull request by bryevdv**
@bokeh/dev if there are smaller docs/examples tasks let's collect them in this one PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ very large or streaming datasets.
 
 Installation
 ------------
-We recommend using the [Anaconda Python distribution](https://www.continuum.io/why-anaconda) and conda to install Bokeh. Enter this command at a Bash or Windows command prompt:
+We recommend using the [Anaconda Python distribution](https://anconda.com/why-anaconda) and conda to install Bokeh. Enter this command at a Bash or Windows command prompt:
```
Comment (mattpap) [inline README.md]: an**a**conda

Comment (bryevdv) [reply README.md]: fixed, thanks

**Comment by bryevdv** [comment]
I don;t know why the integration test is failing it started failing  the branding update commit, which changed no code

**Comment by mattpap** [comment]
> I don;t know why the integration test is failing it started failing the branding update commit, which changed no code.

No clue either. The report is useless. I would suspect a browser version change to be responsible, though looking at metadata, I think it's the same (at last major number).

**Comment by bryevdv** [comment]
I guess we will see in a few minutes when the latest merges to master finish, whether this is a general problem or confined to this branch.

**Comment by mattpap** [comment]
btw. What's up with the last horizontal tick in `plotting/file/custom_datetime_axis`?

image

**Comment by mattpap** [comment]
I compared all packages used for testing in this PR and the last good one, and all are the same.

**Comment by bryevdv** [comment]
Oh, I changed it to use a much simpler tick override mechanism... but that's going to make any extra ticks look stupid... bounds would fix that

**Comment by bryevdv** [comment]
Master is failing now too: https://travis-ci.org/bokeh/bokeh/jobs/269230310 This is a general issue

**Comment by canavandl** [comment]
The failing test (``test_scale_both_resizes_width_and_height_with_fixed_aspect_ratio``) passes locally for me. The issue appears to originate w/ SauceLabs.

**Comment by mattpap** [comment]
It gets even better in https://travis-ci.org/bokeh/bokeh/jobs/269230310 (`master` after PR #6823).

**Comment by mattpap** [comment]
> Yes, it appears SauceLabs has upgraded from Firefox 45 -> Firefox 50.

Metadata doesn't confirm this: https://saucelabs.com/beta/tests/01a511ac16314c7aabde1c48752bb96d/metadata#25.

**Comment by canavandl** [comment]
The Travis logs say so: https://travis-ci.org/bokeh/bokeh/jobs/269213598#L337

**Comment by mattpap** [comment]
OK, that's bloody mess. I think we just have to force version 45, if possible, and later update our tests to work with more modern versions.

**Comment by bryevdv** [comment]
wheres a passing test link, maybe compare metadata against that

**Comment by mattpap** [comment]
We depend on the default version (https://github.com/bokeh/bokeh/blob/master/tests/integration/integration_tests_plugin.py#L65), so this (and a few other references) has to be fixed.

**Comment by bryevdv** [comment]
Also can configure here: https://github.com/bokeh/bokeh/blob/master/tests/integration/integration_tests_plugin.py#L39

But doesn't metadata indicate the version has *not* changed?

**Comment by mattpap** [comment]
> wheres a passing test link, maybe compare metadata against that

> But doesn't metadata indicate the version has not changed?

Yeah, e.g. PR #6823 (sliders styling) also has FF 50 installed, but that's unrelated to sauce labs, because we don't run FF locally.

**Comment by mattpap** [comment]
In the case of lack of ideas to fix this, we can temporarily XFail this test (we have one similar already XFailed). At this point I have no idea what's wrong.

**Comment by bryevdv** [comment]
em either, I will xfail the four tests...

**Comment by bryevdv** [comment]
Not sure if it's coincidence but all four failing tests were the last tests in their respective files.

**Comment by bryevdv** [comment]
passing now will merge when green

**Pull request merged**